### DISCUSSION TITLE
Alternate Titles via ID Cards

### DISCUSSION THREAD

**Discussion by DangerRevolution** [DeltaV-Station/Delta-v#1124] [General]
Gonna make some actual use out of this discussions page 🐙 

Looking to make alternate round-start titles similar to #1037, #1035 and #1036 now that loadouts has been/will be merged.

First stop will be 1035-1037 but after that it's kind of open season so looking for ideas. My first stop will be Station 13 style alternate titles.
just some examples:

Prisoner:
- High Security Prisoner
- Medium Security Prisoner
- Low Security Prisoner

Warden:
- Sheriff
- Brig Chief
- Jailor

Captain:
- Station Overseer
- Station Commander
- (I don't remember the third variant)

These are just hypotheticals, worth discussing, would like to hear ideas :O :)

**Comment by rosieposieeee** [comment]
as cool as corpsman is as a word, it might be nice to have an ungendered alternative, like letting people switch back to 'brigmedic', or possibly adding 'combat medic', 'medical officer', or uh... 'field surgeon'?

**Reply by rosieposieeee** [reply]
and speaking of doctors, how about 'nurse' or 'nurse practicioner', as there is a nurse hat some prefer;

and for psychiatrist, letting them choose 'therapist' instead might be nice

**Reply by IamVelcroboy** [reply]
Corpse-person is the PC term for zombies 😋🧟‍♂️

**Comment by Timemaster99** [comment]
This feature would be best implemented for select civilian roles, like passenger. Command officers can already change their own role name, but it would do very little in the way of improving or changing their roleplay. A "Station Commander" will do the same things as the "Captain", since the role has defined responsibilities that do not change.
On the other hand, roles like Passenger have absolutely no pre-defined direction or responsibility, which means that having options such as "Tourist" and "Off-duty crew member" would _give_ them that direction. 
Prisoners benefit only a marginal amount from this since their role is already defined, but since they have nothing to do except play a character, I can see the more dedicated roleplayers acting out high-security and low-security nicely

**Reply by DangerRevolution** [reply]
>  Command officers can already change their own role name

I think even though they *can* change their title, seeing other options, even if they don't take them could inspire them to change it up, could lead to something interesting, hard to see really without seeing how people treat it in-practice.

**Comment by Timemaster99** [comment]
I played Paradise recently which features multiple titles for _most_ roles, and it's revealed to me some interesting use cases for this system:
- The "lawyer" job by default is an Internal Affairs Agent with an alternate title of Human Resources. Both of these titles could be used for our Lawyer, even considering my Justice PR
- The scientist has multiple specialty titles, like "Xenoarcheologist". Epistemics could make good use of this, since they have three main mechanics (oracle, artifacts, anomalies) to focus on
- Engineer has some mixed titles of little value, like "Maintenance Technician", but then has roles that center on Power generation and construction. To a limited degree, engineers could have a role focusing on construction of new station segments

Of course, they also have roles that suffer from being superfluous (i.g., "Janitor/Custodial Technician", "Chef/Cook", "Atmospheric Technician/Life Support Specialist"). Titles like these should only be used when switching between them can offer a tangible difference in roleplay, but the examples are worth consideration

**Reply by DangerRevolution** [reply]
Baystation style forks such as Paradise are the main inspiration behind this, you'll be seeing a lot of familiar titles soon :)

**Comment by DannyFranklin** [comment]
Station command
